packages/core/src/lib/filter/index.ts

```diff
@@ -302,6 +290,6 @@ class Filter implements IFilter {
 
 export function wakuFilter(
   init: Partial<ProtocolCreateOptions> = {}
-): (components: FilterComponents) => IFilter {
-  return (components: FilterComponents) => new Filter(components, init);
+): (libp2p: Libp2p) => IFilter {
+  return (libp2p: Libp2p) => new Filter(libp2p, init);
```

Comment:
Is it a breaking change?

Reply:
Good call. Updated title to reflect. No need to document as we don't expect users to use this API (yet).